.azure/pipelines/azure-pipelines-mirror-within-azdo.yml

```diff
@@ -0,0 +1,64 @@
+# Configure which branches trigger builds
+trigger:
+  batch: true
+  branches:
+    include:
+    - internal/release/6.0
+
+variables:
+  - group: Mirror-Credentials
+
+# Merges code from one AzDO branch into another
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enableTelemetry: true
+    helixRepo: dotnet/arcade
+    jobs:
+    - job: Merge_Azure_DevOps_Branches
+      enableSBOM: false
+      pool:
+        name: NetCore1ESPool-Internal
+        demands: ImageOverride -equals Build.Server.Amd64.VS2019
+      variables:
+      - name: WorkingDirectoryName
+        value: repo-dir
+      - name: AzdoRepo
+        value: https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore
+      - name: TargetBranchName
+        value: $(Build.SourceBranch)-nonstable
```

Comment:
@dotnet/dnceng will this part do what I'm wanting it to (for the current state, always merge `internal/release/6.0` to `internal/release/6.0-nonstable`

Reply:
I think it would do that, but should there perhaps be some conditions around it (e.g. does it need to look for an "internal" prefix?

Reply:
https://github.com/dotnet/aspnetcore/pull/41715/commits/15aadf21d01aa24ae4a742a2f47053df80366319